hypothesis_bio/hypothesis_bio.py

```diff
@@ -176,44 +179,53 @@ def fasta(draw) -> str:
 
 
 @composite
-def fastq_quality(draw: Callable, size=0) -> str:
+def fastq_quality(
+    draw: Callable, size=0, min_score: int = 0, max_score: int = 62, offset: int = 64
```

Comment:
What's the reasoning behind a max score of 62?

Reply:
Maximum Illumina quality score https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847217/#__sec5title